trace_decoder/src/types.rs

```diff
@@ -6,19 +6,32 @@ use evm_arithmetization::{
 use mpt_trie::nibbles::Nibbles;
 use serde::{Deserialize, Serialize};
 
+/// A block height's type.
 pub type BlockHeight = u64;
+/// A bloom filter's type.
 pub type Bloom = [U256; 8];
+/// A code hash's type.
 pub type CodeHash = H256;
+/// The type of an account address's hash.
 pub type HashedAccountAddr = H256;
+/// The type of a node address's hash.
 pub type HashedNodeAddr = H256;
+/// The type of a storage address's hash.
 pub type HashedStorageAddr = H256;
+/// The type of a hashed storage address"s nibbles.
 pub type HashedStorageAddrNibbles = Nibbles;
+/// The type of a storage address.
 pub type StorageAddr = H256;
+/// The type of a storage address's nibbles.
 pub type StorageAddrNibbles = H256;
+/// A storage value's type.
 pub type StorageVal = U256;
+/// A trie root hash's type.
 pub type TrieRootHash = H256;
+/// Type of a transaction's index within a block.
 pub type TxnIdx = usize;
```

Comment:
Nit: May be better to explicitely describe them as 
```rust
/// A type alias around `foo` for `bar`
```
?

Reply:
Yeah, though we could keep the regular `block height` as you had originally done, instead of putting the actual type name here. as you want!

Reply:
also you can put `u64` between ` to highlight it (it's a primitive type so it can be highlighted in rustdoc)

Reply:
Alright, thank you!